spec/views/curation_concerns/base/_relationships.html.erb_spec.rb

```diff
@@ -4,8 +4,8 @@
   let(:ability) { double }
   let(:solr_doc) { double(id: '123', human_readable_type: 'Work') }
   let(:presenter) { Sufia::WorkShowPresenter.new(solr_doc, ability) }
-  let(:generic_work) { GenericWork.new(id: '456', title: ['Containing work', 'barbaz']) }
```

Comment:
This PR also offers RDF2 support, which means that statistically, the tests that check for `'Containing work'` fail 50% of the time.  As the two titles didn't seem particularly important to the test, the fix was to remove one of them.